build.sbt

```diff
@@ -107,6 +107,7 @@ lazy val amqpTransport = (project in file("rhttpc-amqp")).
     name := "rhttpc-amqp",
     libraryDependencies ++= {
       Seq(
+        "com.typesafe.akka"        %% "akka-http"                     % akkaHttpV,
```

Comment:
Why this change? rhttpc-amqp is client protocol agnostic. It only handle passing requests thru amqp.